manifests/repositories.pp

```diff
@@ -20,8 +20,8 @@
         }
       } else {
         yumrepo { 'htcondor-stable':
-          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${major_release}",
-          baseurl  => "http://research.cs.wisc.edu/htcondor/yum/stable/rhel${major_release}",
+          descr    => "HTCondor Stable RPM Repository for Redhat Enterprise Linux ${facts['os']['release']['major']}",
+          baseurl  => 'http://research.cs.wisc.edu/htcondor/yum/stable/rhel$releasever',
```

Comment:
That's really strange, we are using CentOS 7 here exclusively, and it works... 
Here some info:
```
# python -c 'import yum, pprint; yb = yum.YumBase(); pprint.pprint(yb.conf.yumvar, width=1)'
Loaded plugins: fastestmirror
{'arch': 'ia32e',
 'basearch': 'x86_64',
 'infra': 'stock',
 'releasever': '7',
 'uuid': 'ca7ef8a7-5954-4c42-abf7-379eb3b39c06'}
```
i.e. `releasever` is 7. 
However, I also have: 
```
# cat /etc/redhat-release 
CentOS Linux release 7.3.1611 (Core)
```
But the `releasever` for us comes from here: 
```
# cat /etc/yum.conf | grep distrover
distroverpkg=centos-release
```
So I check:
```
$ yum info centos-release | grep Version
Version     : 7
```
This is an almost unmodified CentOS 7 installation just a few weeks old.